sdk-base/src/main/java/com/mapbox/maps/extension/observable/model/DataSourceType.kt

```diff
@@ -1,38 +1,31 @@
 package com.mapbox.maps.extension.observable.model
 
-import com.google.gson.annotations.SerializedName
-
 /**
  * Describes data source of request for resource-request event.
  */
-enum class DataSourceType {
+enum class DataSourceType(val value: String) {
```

Comment:
The changes made to this class are API breaking. We won't be able to make these changes at this time

Reply:
this goes for all enums in this PR, we need to find a different way for it

Reply:
capturing from chat that adding a ctr is not breaking.